### DISCUSSION TITLE
Hapara Lock Screen Bypass

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#357] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I discovered a way to get past Hapara’s “screen block” which makes use of the fact that Hapara is lazy; the pause screen is actually a force-fullscreened page.
How to:

    Make bookmarks of pages you want to visit beforehand.

    Once your screen is paused, spam unfullscreen. Each time you do, you should see your tabs and bookmark bar come back. Attempt to right-click on the bookmarks bar until a menu shows up.

    In the menu, select “Add folder”. Name it whatever you like. Hit Done. With luck, your tabs and bookmarks should stay at the top of your screen. If not, try again.

    Once the tab and bookmarks bar stays, spam-click one of the bookmarks you made. This lags Hapara into displaying your page instead of the pause page. Turn off your wifi as soon as the page fully loads. This stops data flow between your amd the teacher’s computer, so that they can’t re-pause your screen.

This may and will take several tries to do.

`
Fetched from https://discord.com/channels/419123358698045453/1059260955256684624
`

**Comment by Aka-but-nice** [comment]


**Reply by Aka-but-nice** [reply]
And, why are they all about hapara. I swear, if you start posting stuff about point blank from kajigs that shows you are camping there

**Comment by BackdoorSS** [comment]
> I discovered a way to get past Hapara’s “screen block” which makes use of the fact that Hapara is lazy; the pause screen is actually a force-fullscreened page. How to:
> 
> ```
> Make bookmarks of pages you want to visit beforehand.
> 
> Once your screen is paused, spam unfullscreen. Each time you do, you should see your tabs and bookmark bar come back. Attempt to right-click on the bookmarks bar until a menu shows up.
> 
> In the menu, select “Add folder”. Name it whatever you like. Hit Done. With luck, your tabs and bookmarks should stay at the top of your screen. If not, try again.
> 
> Once the tab and bookmarks bar stays, spam-click one of the bookmarks you made. This lags Hapara into displaying your page instead of the pause page. Turn off your wifi as soon as the page fully loads. This stops data flow between your amd the teacher’s computer, so that they can’t re-pause your screen.
> ```
> 
> This may and will take several tries to do.
> 
> `Fetched from https://discord.com/channels/419123358698045453/1059260955256684624`

I figured this out myself i think its cool just press f11 and it will give you a little time to save the bookmark or like when your playing a gimkit or something.
